TranslucentTB/main.cpp

```diff
@@ -581,8 +599,8 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPreInst, LPSTR pCmdLine, int
 		// Still, we check the --config parameter on the command line, so that should be handled
 		// before the config file. Needs two passes on the command line arguments, lets leave that
 		// for later.
-		ParseCmdOptions(); //command line argument settings
 		ParseConfigFile(L"config.cfg"); //config file settings
```

Comment:
Do we want to parse config.cfg no matter what the user has specified? It would be better to change ParseCmdOptions() to parse config.cfg if it didn't find any other --config option specified. You could add a check for configfile.empty() after the first loop in ParseCmdOptions() and load config.cfg there.